Formula/libvisual-0.4.rb

```diff
@@ -0,0 +1,23 @@
+class Libvisual04 < Formula
+  desc "Audio Visualization"
+  homepage "https://github.com/Libvisual/libvisual"
+  url "https://github.com/Libvisual/libvisual/releases/download/libvisual-0.4.2/libvisual-0.4.2.tar.gz"
+  sha256 "63085fd9835c42c9399ea6bb13a7ebd4b1547ace75c4595ce8e9759512bd998a"
+  license "LGPL-2.1-or-later"
+  # head "https://github.com/Libvisual/libvisual", branch: "0.4.x"
```

Comment:
Either remove this line or use the `master` branch.

Reply:
@carlocab will drop it then. Please nopte that 0.4.x is not an accident and that master is no option here, because of a completely different build system, i.e. CMake versus GNU Autotools.

Reply:
That's fine, we can use CMake for `--HEAD` builds if desired. `head` is for the latest development version. But dropping it for now seems simpler, so let's do that.